corehq/apps/domain/calculations.py

```diff
@@ -261,8 +249,6 @@ def not_implemented(domain, *args):
     'last_form_submission', 'has_app', 'web_users', 'active_apps',
     'uses_reminders', 'sms--I', 'sms--O', 'sms_in_last', 'sms_in_last--30',
     'sms_in_last_bool', 'sms_in_last_bool--30', 'sms_in_in_last--30',
-    'sms_out_in_last--30', 'j2me_forms_in_last--30', 'j2me_forms_in_last--60',
```

Comment:
Was `'sms_out_in_last--30'` meant to be removed?

Reply:
I think that got caught in my j2me drag net. I will revert that